{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/core/views.py

```diff
@@ -128,6 +134,38 @@ def reset_password(request, *args, **kwargs):
     return Response(response_data, status=status.HTTP_200_OK)
 
 
+@api_view(["get"])
+@permission_classes([permissions.AllowAny])
+def request_reset_code(request, *args, **kwargs):
+    email = kwargs.get("email")
+    user = User.objects.filter(email=email).first()
+    if not user:
+        raise ValidationError(detail={"non_field_errors": ["User not found with that email"]})
+    UserResetPasswordCode.objects.create_code(user=user, code=random_pin_generator(count=5))
+    return Response(status=status.HTTP_204_NO_CONTENT)
+
+
+@api_view(["post"])
+@permission_classes([permissions.AllowAny])
+def reset_password_with_code(request, *args, **kwargs):
+    email = kwargs.get("email")
+    user = User.objects.filter(email=email).first()
+    if not user:
+        raise ValidationError(detail={"non_field_errors": ["User not found with that email"]})
+    serializer = CodeResetPasswordSerializer(data=request.data, context={"user": user})
+    serializer.is_valid(raise_exception=True)
+
+    code_from_db = serializer.context.get("code_from_db")
+    code_from_db.is_used = True
+    code_from_db.save()
+
+    user.set_password(request.data.get("password"))
+    user.save()
+
+    u = UserLoginSerializer.login(user, request)
+    return Response(status=status.HTTP_200_OK, data=u)
```

Comment:
This is a lot of code to add for something that is only needed for mobile app builds, and we aren't able to remove it with a cookiecutter flag.
I'd like to discuss.